docs/dev/table/catalogs.md

```diff
@@ -144,7 +144,7 @@ List<String> tables = catalog.listTables("mydb); // tables should contain "mytab
 ## Catalog API
 
 Note: only catalog program APIs are listed here. Users can achieve many of the same funtionalities with SQL DDL. 
-For detailed DDL information, please refer to [SQL DDL](https://ci.apache.org/projects/flink/flink-docs-release-1.9/dev/table/sql.html#ddl).
+For detailed DDL information, please refer to [SQL DDL](sql/ddl.html).
```

Comment:
Should the link here be `/dev/table/sql/ddl.html`

Reply:
relative link also works here.